src/main/java/com/asana/examples/ExampleCreateProjectAndStreamEvents.java

```diff
@@ -8,16 +8,13 @@
 import java.util.List;
 
 /**
- *
  * API Key Instructions:
- *
+ * <p/>
```

Comment:
This is debatable but technically correct. Down with taking it out. http://www.oracle.com/technetwork/java/javase/documentation/index-137868.html#format